be/src/runtime/tablets_channel.cpp

```diff
@@ -228,17 +241,24 @@ Status TabletsChannel::_open_all_writers(const PTabletWriterOpenRequest& request
     }
     if (index_slots == nullptr) {
         std::stringstream ss;
-        ss << "unknown index id, key=" << _key;
+        ss << "unknown index id, key=" << _key << " tablet_id=" << tablet_id;
         return Status::InternalError(ss.str());
     }
-    for (auto& tablet : request.tablets()) {
+    int64_t partition_id = _tablet_partition_map[tablet_id];
+    DCHECK(partition_id != 0);
+    auto tablets = _partition_tablets_map[partition_id];
+    DCHECK(tablets.size() > 0);
+    LOG(INFO) << fmt::format(
```

Comment:
DEBUG level is enough, high frequency ingestions would leads to too many logs.

Reply:
done